oteapi_dlite/strategies/oceanlab_influx_parser.py

```diff
@@ -0,0 +1,279 @@
+"""Strategy for oceanlab data parsing from Influx DB."""
+
+import sys
+from typing import Annotated, Optional
+
+import cachetools  # type: ignore
+import dlite
+import influxdb_client
+import jinja2
+from fastapi import logger
```

Comment:
Didn't know this was a thing! 😮

Reply:
there were other loggers as well. But this works

Reply:
changed to normal python logging

Reply:
excellent